purchase_manual_add_supplierinfo/tests/test_purchase_manual_add_supplierinfo.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, SavepointCase
+
+
+class TestPurchaseManualAddSupplierinfo(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.partner = cls.env.ref("base.res_partner_1")
+        cls.product_6 = cls.env.ref("product.product_product_6")
+        cls.product_7 = cls.env.ref("product.product_product_7")
+
+    def test_update_supplierinfo_from_purchase(self):
+        products = self.product_6 + self.product_7
+        products.write({"seller_ids": [(6, 0, [])]})
+        purchase_order = self._create_purchase_order(self.partner, products)
+        purchase_order.button_confirm()
+        self.assertEqual(purchase_order.all_supplierinfo_ok, False)
```

Comment:
Never use 

 `self.assertEqual(purchase_order.all_supplierinfo_ok, False)`

But use

self.assertFalse(purchase_order.all_supplierinfo_ok)`

Reply:
@sebastienbeau , it's done.